Sources/LiveKit/Core/Engine.swift

```diff
@@ -70,14 +66,15 @@ internal class Engine: MulticastDelegate<EngineDelegate> {
         ConnectivityListener.shared.add(delegate: self)
 
         // trigger events when state mutates
-        self.state.onMutate = { [weak self] oldState, newState in
+        self.state.onMutate = { [weak self] state, oldState in
```

Comment:
ohh was the order the other way around?

Reply:
I just changed the ordering in this PR 🙂